frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.unit.spec.tsx

```diff
@@ -128,4 +129,20 @@ describe("DashboardApp", function () {
       expect(mockEvent.returnValue).toBe(undefined);
     });
   });
+
+  describe("empty dashboard", () => {
+    it("should prompt the user to add a question if they have write access", async () => {
+      await setup();
+
+      expect(screen.getByText(/add a saved question/i)).toBeInTheDocument();
+    });
+
+    it("should should show an empty state without a prompt to add a question if the user doesn't have write access", async () => {
```

Comment:
nit

```suggestion
    it("if the user doesn't have write access, should should show an empty state without an 'add a question' prompt", async () => {
```

Reply:
Cleaned up the wording a bit, although I kept it starting with `should` so it could follow the `it should` naming pattern